### PR TITLE
Merge dev (version 1.1.0) into master

### PR DESCRIPTION
**Pull request by JIceberg**
# Version 1.1.0

The PR to merge 1.1.0 into master. This will cause breaking changes. These will need to be fixed before release after documentation is finished.

### CODE REVIEW

**Comment by JIceberg** [comment]
Merge conflicts removed. Ready to merge when documentation is complete.

File: core/src/main/java/com/arcrobotics/ftclib/controller/PIDFController.java [hunk 1]
```diff
@@ -74,53 +74,6 @@ public void reset() {
...
-    /**
-     * Implements a control calculation onto the affected motor
-     *
-     * @param affected The affected motor of the mechanism
-     */
-    public void control(Motor affected){
-        control(affected, prevErrorVal);
-    }
-
-
```
Comment (JIceberg) [inline core/src/main/java/com/arcrobotics/ftclib/controller/PIDFController.java]: This was incredibly useless and confusing regardless; PID control should've been internal for the motors.

**Review by JIceberg — COMMENTED** [review]
Let's get this merged by Thursday.

File: examples/src/main/java/com/example/ftclibexamples/CommandSample/SampleTeleOp.java [hunk 2]
```diff
@@ -1,22 +1,50 @@
...
+import com.qualcomm.robotcore.eventloop.opmode.Disabled;
 import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
 
 @TeleOp(name="Sample TeleOp")
-public class SampleTeleOp extends LinearOpMode {
+@Disabled
+public class SampleTeleOp extends CommandOpMode {
 
-    SampleRobot m_robot = new SampleRobot(this, true);
+    static final double WHEEL_DIAMETER = 100.0; // centimeters
```
Comment (puravdatta-sudo) [inline examples/src/main/java/com/example/ftclibexamples/CommandSample/SampleTeleOp.java]: This needs to be millimeters, not centimeters

Comment (puravdatta-sudo) [inline examples/src/main/java/com/example/ftclibexamples/CommandSample/SampleTeleOp.java]: This needs to be mm not cm

File: examples/src/main/java/com/example/ftclibexamples/PurePursuitSample.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.Disabled;
+
+@Autonomous
+@Disabled
+public class PurePursuitSample extends CommandOpMode {
+
+    // define our constants
+    static final double TRACKWIDTH = 13.7;
+    static final double TICKS_TO_INCHES = 15.3;
```
Comment (puravdatta-sudo) [inline examples/src/main/java/com/example/ftclibexamples/PurePursuitSample.java]: This should be computed, not a static variable.

**Review by puravdatta-sudo — CHANGES_REQUESTED** [review]

**Review by JarnaChao09 — APPROVED** [review]

File: examples/src/main/java/com/example/ftclibexamples/PurePursuitSample.java (hunk 3 above)
Comment (JIceberg) [reply examples/src/main/java/com/example/ftclibexamples/PurePursuitSample.java]: image
Does something like this work?

Comment (puravdatta-sudo) [reply examples/src/main/java/com/example/ftclibexamples/PurePursuitSample.java]: Yup, another thing which could be done is just having wheel rad and ticks and then figure it out based on that.

**Review by puravdatta-sudo — APPROVED** [review]
Looks good to me.

**Review by Lunerwalker2 — APPROVED** [review]

File: core/src/main/java/com/arcrobotics/ftclib/geometry/Rotation2d.java [hunk 4]
```diff
@@ -38,6 +38,8 @@ public Rotation2d() {
      * @param value The value of the angle in radians.
      */
     public Rotation2d(double value) {
+        while (value > Math.PI) value -= 2 * Math.PI;
```
[the remark this answers is not in the document]
Comment (JIceberg) [reply core/src/main/java/com/arcrobotics/ftclib/geometry/Rotation2d.java]: Nah. Doesn't really fit the implementation.

**Review by Jaiming724 — APPROVED** [review]

**Pull request merged**